types/qaengine/qagrpc/fetchanswer.pb.go

```diff
@@ -64,6 +64,7 @@ type Problem struct {
 	Hints       []string `protobuf:"bytes,4,rep,name=hints,proto3" json:"hints,omitempty"`
 	Options     []string `protobuf:"bytes,5,rep,name=options,proto3" json:"options,omitempty"`
 	Default     []string `protobuf:"bytes,6,rep,name=default,proto3" json:"default,omitempty"`
+	Pattern string `protobuf:"bytes,7,rep,name=pattern,proto3" json:"pattern,omitempty"`
```

Comment:
We should not be editing this directly. Only the proto file should be edited and this should be generated.